.codecov.yml

```diff
@@ -0,0 +1,19 @@
+codecov:
+  notify:
+    require_ci_to_pass: yes
+
+coverage:
+  precision: 1
+  round: nearest
+  range: '10...100'
+
+  status:
```

Comment:
configuration for statuses from codecov:
- only on prs
- only fails if coverage drops by 25%